File: Benchmarks/enterprise/java/jakarta/testcode/BenchmarkTest48870.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import jakarta.ws.rs.*;
import jakarta.ws.rs.core.Context;
import jakarta.ws.rs.core.MediaType;
import jakarta.ws.rs.core.Response;

@Path("/")
public class BenchmarkTest48870 {

    @GET
    @Path("/BenchmarkTest48870")
    @Produces(MediaType.APPLICATION_JSON)
    public Response BenchmarkTest48870(@CookieParam("session_token") String sessionToken, @Context HttpServletRequest request, @Context HttpServletResponse response) throws Exception {
        String cookieValue = sessionToken != null ? sessionToken : "";
        String data = cookieValue.isEmpty() ? "default" : cookieValue;
        response.setContentType("application/json");
        return Response.ok("{\"echo\":\"" + data + "\"}", MediaType.APPLICATION_JSON).build();
    }
}

File: Benchmarks/enterprise/python/django/testcode/benchmark_test_48870.py
# SPDX-License-Identifier: Apache-2.0
from django.http import JsonResponse


def BenchmarkTest48870(request, path_param):
    path_value = path_param
    data = '%s' % (path_value,)
    return JsonResponse({'status': 'ok'}, status=200, headers={'Content-Language': str(data)})
